packages/components/src/components/primitives/Textarea/Textarea.stories.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { Meta } from '@storybook/react';
+import { Textarea } from './Textarea';
+import { HStack, StackDivider, VStack } from '@chakra-ui/layout';
+import { textareaSizes } from './types';
+
+export default {
+  component: Textarea,
+  title: 'Components/Primitives/Textarea',
+  argTypes: {
+    size: {
+      options: textareaSizes,
+      control: { type: 'select' },
+    },
+    isInvalid: {
+      control: { type: 'boolean' },
+    },
+    isDisabled: {
+      control: { type: 'boolean' },
+    },
```

Comment:
nit: this is not necessary, storybook detects and handle boolean values

Reply:
done!